src/shared/components/checkbox/Checkbox.tsx

```diff
@@ -19,8 +19,11 @@ import classNames from 'classnames';
 
 import styles from './Checkbox.scss';
 
+type Theme = 'lighter' | 'light' | 'dark';
+
 type ClassNames = Partial<{
   checkboxHolder: string;
+  default: string;
   checked: string;
   unchecked: string;
   disabled: string;
```

Comment:
A big question for me now is whether we need to pass css class names to the check component at all, or its relevant styles can be overridden entirely through css variables.

Reply:
I agree. This would make it easier to customise the checkboxes. I'll replace these classes with CSS variables.

Reply:
✅